src/components/config/index.ts

```diff
@@ -191,9 +192,9 @@ export const setupConfig = async (flags: any): Promise<void> => {
     flags.postman === undefined &&
     flags.insomnia === undefined
   ) {
-    throw new Error(
-      'Configuration file not found. By default, Monika looks for monika.yml configuration file in the current directory.\n\nOtherwise, you can also specify a configuration file using -c flag as follows:\n\nmonika -c <path_to_configuration_file>\n\nYou can create a configuration file via web interface by opening this web app: https://hyperjumptech.github.io/monika-config-generator/'
-    )
+    const configFileName = 'monika.yml'
```

Comment:
We can put this static variable as constant variable like `DEFAULT_CONFIG_INTERVAL`.